Automattic/jetpack
# Update hardcoded plan slug

Update hardcoded plan slug after making Personal a legacy wpcom plan

Fixes #

#### Changes proposed in this Pull Request:

We want the Pro plan to be visible here instead of Personal:

<img width="747" alt="Screenshot 2022-04-01 at 12 26 18" src="https://user-images.githubusercontent.com/82778/161236094-cd38933a-3f64-4375-b10a-d0fa998068e1.png">

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






1. With a test site on the free plan, on WordPress.com
2. Add a new Payment Button block
3. See a notice with the Pro plan

Before:
<img width="727" alt="Screenshot 2022-04-01 at 12 29 24" src="https://user-images.githubusercontent.com/82778/161236550-f0a17f32-f7d0-4cf6-ae4e-6ed512e31d79.png">

After:
<img width="747" alt="Screenshot 2022-04-01 at 12 26 18" src="https://user-images.githubusercontent.com/82778/161236094-cd38933a-3f64-4375-b10a-d0fa998068e1.png">
